### PR TITLE
match current dark mode designs

### PR DESCRIPTION
**Pull request by webb04**
## Changes

- Use inverted over kicker in dark mode 
- Darken the border colour on links in dark mode
- Analysis headline dark styles

## Screenshots

| Before | After |
| --- | --- |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |

### CODE REVIEW

File: src/components/anchor.tsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ const styles = css`
 
     ${darkModeCss`
         color: ${neutral[86]};
+        border-bottom: 0.0625rem solid ${neutral[46]};
```
Comment (JamieB-gu) [inline src/components/anchor.tsx:27]: Is this solvable with just `border-color`?

File: src/components/figCaption.tsx [hunk 2]
```diff
@@ -19,15 +20,16 @@ interface TriangleProps {
     format: Format;
 }
 
-const triangleStyles = (colour: string): SerializedStyles => css`
+const triangleStyles = (colour: string, inverted?: string): SerializedStyles => css`
     fill: ${colour};
     height: 0.8em;
     padding-right: ${remSpace[1]};
+    ${inverted ? darkModeCss`fill: ${inverted};` : ``}
```
Comment (JamieB-gu) [inline src/components/figCaption.tsx:27]:
```suggestion
    ${inverted ? darkModeCss`fill: ${inverted};` : ''}
```

File: src/renderer.ts [hunk 3]
```diff
@@ -42,20 +42,26 @@ const getAttr = (attr: string) => (node: Node): Option<string> =>
...
-    display: inline-block;
-
-    &::before {
-        content: '';
-        background-color: ${colour};
-        width: 1rem;
-        height: 1rem;
-        border-radius: .5rem;
+const bulletStyles = (format: Format): SerializedStyles => {
+const { kicker, inverted } = getPillarStyles(format.pillar);
```
Comment (JamieB-gu) [inline src/renderer.ts:46]:
```suggestion
    const { kicker, inverted } = getPillarStyles(format.pillar);
```

**Comment by benwuersching** [comment]
I'm a bit late as it's already merged. In any case, this looks all good @webb04. Thank you.

**Pull request merged**